test/e2e/framework/framework.go

```diff
@@ -66,11 +66,28 @@ type Framework struct {
 
 // New setups a testing framework using a kubeconfig path and the game server image to use for testing.
 func New(kubeconfig string) (*Framework, error) {
+	return newFramework(kubeconfig, 0, 0)
+}
+
+// NewForLoadTesting setups a testing framework using a kubeconfig path and the game server image
+// to use for load testing with QPS and Burst overwrites.
+func NewForLoadTesting(kubeconfig string, qps float32, burst int) (*Framework, error) {
```

Comment:
```suggestion
func NewWithRates(kubeconfig string, qps float32, burst int) (*Framework, error) {
```
^ Something like that? That explains what it does, rather than just the first place it gets used? WDYT? That was the best name I could come up with.

Reply:
Ok, agree with naming